scripts/Iris.json

```diff
@@ -1,8 +1,7 @@
 {
-    "citation": "R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.
-
-",
+    "citation": "R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.",
     "description": "Famous dataset from R. A. Fisher",
+    "addendum": "This dataset has been corrected. Information Source: Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.",
```

Comment:
Thanks @zhangcandrew. I'd completely forgotten that we had an `addendum` category. My initial thought on its continued use is that since there is no mechanism for exposing that information to the user that we're better off putting the information in the description. What do you think?

 `addendum` is also not in the datapackage spec.